docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  jenkins:
+    image: jenkins/jenkins:lts
+    user: 1000:1000 #set user to root to avoid permission issues
```

Comment:
This sets user to jenkins instead of root.

Reply:
sure I'll update it something like this:

```
user: jenkins:jenkins
```
I think this would work

Reply:
Sure!